### PR TITLE
Add Label 22 Takeoff Report

### PR DESCRIPTION
**Pull request by makrsmark**
- renamed Label_22 to Label_22_POS and added preambles

### CODE REVIEW

File: lib/plugins/Label_22_OFF.test.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    expect(decodeResult.raw.arrival_icao).toBe('KPWM');
+    expect(decodeResult.formatted.items.length).toBe(7);
+    expect(decodeResult.formatted.items[0].label).toBe('Flight Number');
+    expect(decodeResult.formatted.items[0].value).toBe('YX3661');
+    expect(decodeResult.formatted.items[1].label).toBe('Departure Day');
+    expect(decodeResult.formatted.items[1].value).toBe('25');
+    expect(decodeResult.formatted.items[2].label).toBe('Arrival Day');
+    expect(decodeResult.formatted.items[2].value).toBe('25');
+    expect(decodeResult.formatted.items[3].label).toBe('Message Timestamp');
+    expect(decodeResult.formatted.items[3].value).toBe('17:12:00');
```
Comment (makrsmark) [inline lib/plugins/Label_22_OFF.test.ts:42]: this is a little weird, but what I'm calling the timestamp has minute resolution, but the actual takeoff time has second resolution. Open to ideas

File: lib/plugins/Label_22_OFF.test.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    expect(decodeResult.raw.arrival_icao).toBe('EPRZ');
+    expect(decodeResult.formatted.items.length).toBe(8);
+    expect(decodeResult.formatted.items[0].label).toBe('Flight Number');
+    expect(decodeResult.formatted.items[0].value).toBe('XA0000');
+    expect(decodeResult.formatted.items[1].label).toBe('Aircraft Position');
+    expect(decodeResult.formatted.items[1].value).toBe('38.947 N, 77.435 W');
+    expect(decodeResult.formatted.items[2].label).toBe('Day of Month');
+    expect(decodeResult.formatted.items[2].value).toBe('25');
+    expect(decodeResult.formatted.items[3].label).toBe('Message Timestamp');
+    expect(decodeResult.formatted.items[3].value).toBe('11:52:00');
```
Comment (makrsmark) [inline lib/plugins/Label_22_OFF.test.ts:72]: in this variant, both the timestamp and takeoff time have minute resolution - i could be parsing backwards but it probably doesn't matter

File: lib/plugins/Label_22_OFF.test.ts (hunk 1 above)
Comment (kevinelliott) [reply lib/plugins/Label_22_OFF.test.ts:42]: I would say that as long as the raw value is still defined with seconds, the formatted can be whatever.

**Review by kevinelliott — APPROVED** [review]
LGTM!

**Pull request merged**